cdk.json

```diff
@@ -31,36 +31,32 @@
       "aws-cn"
     ],
     "dev": {
-      "IMAGE_PATH_AND_TAG": "ghcr.io/sage-bionetworks/myapp:release-1.0",
+      "IMAGE_PATH_AND_TAG": "ghcr.io/sage-bionetworks/synapse-docker-registry:dev",
       "AWS_DEFAULT_REGION": "us-east-1",
-      "PORT": "3838",
+      "PORT": "443",
       "CONTAINER_ENV": {
-        "name1": "value1",
-        "name2": "value2"
       },
       "TAGS": {
-        "CostCenter": "NO PROGRAM / 000000",
-        "OwnerEmail": "joe@sagebase.org"
+        "CostCenter": "Platform Infrastructure / 990300",
+        "OwnerEmail": "synapseeng@sagebase.org"
       },
-      "STACK_NAME_PREFIX": "myapp-dev",
-      "VPC_CIDR": "172.31.0.0/24",
-      "ACM_CERT_ARN": "arn:aws:acm:us-east-1:<DEV_ACCOUNT_ID>:certificate/<DEV_CERT_ID>"
+      "STACK_NAME_PREFIX": "registry-dev",
+      "VPC_CIDR": "172.30.0.0/24",
```

Comment:
this IP is already being used. please select a different one. instructions here https://sagebionetworks.jira.com/wiki/spaces/IT/pages/2850586648/Setup+AWS+VPC

<img width="1344" alt="Screenshot 2024-09-27 at 10 59 50 AM" src="https://github.com/user-attachments/assets/d68fd6cd-5a3d-417a-aba8-14f5de50c6c1">

Reply:
Updated to 172.29.0.0/24, which shows 'no matches' in the VPC IP Address Manager.